src/components/ProjectSearch/TextSearch.js

```diff
@@ -17,7 +17,18 @@ import { getRelativePath } from "../../utils/sources-tree";
 import { highlightMatches } from "./textSearch/utils/highlight";
 import { statusType } from "../../reducers/project-text-search";
 
-export default class TextSearch extends Component {
+type Props = {
+  sources: Object,
+  results: Object,
```

Comment:
Any reason this is not `results: Array,`?

Reply:
Nope, sorry about that, oversight by me. Change committed.